src/app/home/layouts/budgets-form/budgets-form.component.ts

```diff
@@ -1,6 +1,8 @@
 import {Component, OnInit} from '@angular/core';
 import {InitialSettingsService} from '../../cards/initial-settings/initial-settings.service';
 import {basicAnimation} from '../../../shared/animation';
+import {NgForm} from '@angular/forms';
+import {Categories, CategoryDto, CyclesDict, CyclesDto} from './dtos';
```

Comment:
Why are those capitalized? I assume those are just data objects, and in that case, it should follow the camel case convention. Capitalized names are mostly used for classes, interfaces and types.